node/JobRoutes.js

```diff
@@ -0,0 +1,7 @@
+app.get('/JobsTable', async function (req, res) { 
+    console.log('Request processed'); 
+    const response = await fetch('http://localhost:8080/api/jobs/jobRoles',{method:'GET',headers:{}})
```

Comment:
The server should be configurable. You have no guarantee the customer would have the front- and back-ends deployed on the same system.